app/src/androidTest/java/org/simple/clinic/teleconsultlog/teleconsultrecord/TeleconsultRecordTest.kt

```diff
@@ -92,39 +83,19 @@ class TeleconsultRecordTest {
         )
     )
 
-    teleconsultRecordDao.save(
-        listOf(
-            teleconsultRecord1,
-            teleconsultRecord2
-        )
-    )
-
-    val teleconsultRecordPrescribedDrug1 = TestData.teleconsultationRecordPrescribedDrug(
-        teleconsultRecordId = teleconsultRecordId1,
-        prescribedDrugUuid = prescribedDrugUuid,
-    )
-
-    val teleconsultRecordPrescribedDrug2 = TestData.teleconsultationRecordPrescribedDrug(
-        teleconsultRecordId = teleconsultRecordId1,
-        prescribedDrugUuid = prescribedDrugUuid2,
-    )
-
-    val expectedTeleconsultRecordPrescribedDrugList = listOf(
-        teleconsultRecordPrescribedDrug2,
-        teleconsultRecordPrescribedDrug1
+    val teleconsultRecordList = listOf(
+        teleconsultRecord1,
+        teleconsultRecord2
     )
 
-    teleconsultRecordPrescribedDrugDao.save(
-        expectedTeleconsultRecordPrescribedDrugList
+    teleconsultRecordDao.save(
+        teleconsultRecordList
     )
 
     // when
-    val teleconsultRecordWithPrescribedDrugs = teleconsultRecordWithPrescribedDrugs
-        .getCompleteTeleconsultLog(teleconsultRecordId1)
+    val expectedTeleconsultRecordList = teleconsultRecordDao.getAll()
```

Comment:
expected list would be our current list, so it would check something like : 
`val teleconsultRecordListReceived = teleconsultRecordDao.getAll()` 
`assertThat(teleconsultRecordListReceived).isEqualTo(expectedTeleconsultRecordList)`

Reply:
Yup, seems right.